### DISCUSSION TITLE
xterm.js in output folder

### DISCUSSION THREAD

**Discussion by Nuthin2do** [CodinGame/monaco-vscode-api#518] [Q&A] [answered]
Hi,

when i build my solution for production then in the js-folder appears a xterm.js and some addons of it like this ->

<img width="140" alt="image" src="https://github.com/user-attachments/assets/b0dd9a27-6c51-4d11-828d-2c01e721536c">

Why is a xterm.js generated? Or better why is the xterm.js (and its addons) permanently generated even though i threw any of your packages away and installed only the vscode and monaco package? ->

<img width="422" alt="image" src="https://github.com/user-attachments/assets/360d8725-3e4f-49e6-9c10-330ca64ce3d8">

Is it possible to exclude xterm.js and have an absolut mnimal output for the editor?

Thanks in advance.

**Comment by CGNonofr** [comment, marked as answer]
Thanks for your report, it seems to be a regression of the last version :+1:

**Reply by CGNonofr** [reply]
fixed in https://github.com/CodinGame/monaco-vscode-api/pull/520
